OrbitGl/AccessibleTimeGraph.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "AccessibleTimeGraph.h"
+
+#include "GlCanvas.h"
+#include "TimeGraph.h"
+
+AccessibilityRect TimeGraphAccessibility::AccessibleLocalRect() const {
+  auto canvas = time_graph_->GetCanvas();
+  return AccessibilityRect(0, 0, canvas->GetWidth(), canvas->GetHeight());
+}
+
+AccessibilityState TimeGraphAccessibility::AccessibleState() const {
+  using State = orbit_gl::AccessibilityState;
+  return State::Focusable;
```

Comment:
nit. is the indirection needed? "return orbit_gl::AccessibilityState::Focusable"?

Reply:
No, it's not, thanks. The result was composed of more than one state in the past, now this doesn't make sense any more.